### PR TITLE
Initial import of Mexico, the 31 states and 1 federal district, and …

### PR DESCRIPTION
**Pull request by jdmgoogle**
…300 federal electoral districts.

### CODE REVIEW

**Comment by jdmgoogle** [comment]
Friendly ping to @djbridges and @jpmckinney. In general, what's the process to get these reviewed?

File: identifiers/country-mx.csv [hunk 1]
```diff
@@ -0,0 +1,334 @@
+id,name
+ocd-division/country:mx,México
```
Comment (ccongchen) [inline identifiers/country-mx.csv]: For consistency please use 'Mexico' instead.

File: identifiers/country-mx/districts.csv [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+ocd-division/country:mx/state:ja/fed:12,XII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:13,XIII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:14,XIV Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:15,XV Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:16,XVI Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:17,XVII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:18,XVIII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:19,XIX Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:20,XX Federal Electoral District of Jalisco
+ocd-division/country:mx/state:em/fed:1,I Federal Electoral District of México
```
Comment (ccongchen) [inline identifiers/country-mx/districts.csv]: For consistency, please use 'Mexico' instead.

File: identifiers/country-mx/districts.csv [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+ocd-division/country:mx/state:em/fed:32,XXXII Federal Electoral District of México
+ocd-division/country:mx/state:em/fed:33,XXXIII Federal Electoral District of México
+ocd-division/country:mx/state:em/fed:34,XXXIV Federal Electoral District of México
+ocd-division/country:mx/state:em/fed:35,XXXV Federal Electoral District of México
+ocd-division/country:mx/state:em/fed:36,XXXVI Federal Electoral District of México
+ocd-division/country:mx/state:em/fed:37,XXXVII Federal Electoral District of México
+ocd-division/country:mx/state:em/fed:38,XXXVIII Federal Electoral District of México
+ocd-division/country:mx/state:em/fed:39,XXXIX Federal Electoral District of México
+ocd-division/country:mx/state:em/fed:40,XL Federal Electoral District of México
+ocd-division/country:mx/state:em/fed:41,XLI Federal Electoral District of México
```
Comment (ccongchen) [inline identifiers/country-mx/districts.csv]: For line 126-166, please use "Mexico" instead.

File: identifiers/country-mx/states.csv [hunk 4]
```diff
@@ -0,0 +1,34 @@
+id,name
+ocd-division/country:mx,México
```
Comment (ccongchen) [inline identifiers/country-mx/states.csv]: Are there any particular reason that we use 'Mexico City' for the state name (without the accent) but 'México' for the country name (with the accent)?

File: identifiers/country-mx/districts.csv [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+ocd-division/country:mx/state:cs/fed:4,IV Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:5,V Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:6,VI Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:7,VII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:8,VIII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:9,IX Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:10,X Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:11,XI Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:12,XII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:13,XIII Federal Electoral District of Chiapas
```
Comment (ccongchen) [inline identifiers/country-mx/districts.csv]: According to https://en.wikipedia.org/wiki/Federal_electoral_districts_of_Mexico, we only have 12 fed districts in Chiapas.

File: identifiers/country-mx.csv (hunk 1 above)
Comment (jdmgoogle) [reply identifiers/country-mx.csv]: Source data was inconsistent with accents.

File: identifiers/country-mx/districts.csv (hunk 5 above)
Comment (ccongchen) [reply identifiers/country-mx/districts.csv]: Your list does match with the INE list http://cartografia.ife.org.mx/sige7/?distritacion=federal though, which I assume is the authoritative source for this.

File: identifiers/country-mx/districts.csv [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+ocd-division/country:mx/state:ja/fed:13,XIII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:14,XIV Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:15,XV Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:16,XVI Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:17,XVII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:18,XVIII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:19,XIX Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:20,XX Federal Electoral District of Jalisco
+ocd-division/country:mx/state:em/fed:1,I Federal Electoral District of Mexico
+ocd-division/country:mx/state:em/fed:2,II Federal Electoral District of Mexico
```
Comment (ccongchen) [inline identifiers/country-mx/districts.csv]: This is listed as "México" on the INE site, would it be better to match?

File: identifiers/country-mx/districts.csv [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+ocd-division/country:mx/state:co/fed:2,II Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:3,III Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:4,IV Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:5,V Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:6,VI Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:7,VII Federal Electoral District of Coahuila
+ocd-division/country:mx/state:cl/fed:1,I Federal Electoral District of Colima
+ocd-division/country:mx/state:cl/fed:2,II Federal Electoral District of Colima
+ocd-division/country:mx/state:df/fed:1,I Federal Electoral District of Mexico City
+ocd-division/country:mx/state:df/fed:2,II Federal Electoral District of Mexico City
```
Comment (ccongchen) [inline identifiers/country-mx/districts.csv]: This is listed as 'Ciudad de México', would it be better to match?

File: identifiers/country-mx.csv [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+ocd-division/country:mx/state:cs/fed:13,XIII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:2,II Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:3,III Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:4,IV Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:5,V Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:6,VI Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:7,VII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:8,VIII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:9,IX Federal Electoral District of Chiapas
+ocd-division/country:mx/state:df,Mexico City
```
Comment (ccongchen) [inline identifiers/country-mx.csv]: Line 57-81 'Mexico City' is listed as 'Ciudad de México' on INE site, would it be better to match?

File: identifiers/country-mx/states.csv [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+ocd-division/country:mx,Mexico
+ocd-division/country:mx/state:ag,Aguascalientes
+ocd-division/country:mx/state:bc,Baja California
+ocd-division/country:mx/state:bs,Baja California Sur
+ocd-division/country:mx/state:cm,Campeche
+ocd-division/country:mx/state:cs,Chiapas
+ocd-division/country:mx/state:ch,Chihuahua
+ocd-division/country:mx/state:co,Coahuila
+ocd-division/country:mx/state:cl,Colima
+ocd-division/country:mx/state:df,Mexico City
```
Comment (ccongchen) [inline identifiers/country-mx/states.csv]: 'Mexico City' is listed as 'Ciudad de México' on INE site, would it be better to match?

File: identifiers/country-mx/districts.csv (hunk 7 above)
Comment (jdmgoogle) [reply identifiers/country-mx/districts.csv]: To the extent possible that we can keep all the content in a single language, I'd like to do that. End-users who want to translate this data into other languages (E.g., Spanish) could do so without us mixing languages in the source feed.

File: identifiers/country-mx.csv (hunk 8 above)
Comment (jdmgoogle) [reply identifiers/country-mx.csv]: See above comment.

File: identifiers/country-mx/states.csv (hunk 9 above)
Comment (jdmgoogle) [reply identifiers/country-mx/states.csv]: See above comment.

File: identifiers/country-mx/districts.csv (hunk 7 above)
Comment (ccongchen) [reply identifiers/country-mx/districts.csv]: SG! It is reasonable to have a single language for sure. (nit: we may want to remove the accents for some of the state names, knowing that it is relatively painless for English speakers as well.)

File: identifiers/country-mx/districts.csv (hunk 6 above)
Comment (ccongchen) [reply identifiers/country-mx/districts.csv]: In the context of our discussions on language(s), English sounds perfect in this case!

File: identifiers/country-mx.csv [hunk 10]
```diff
@@ -0,0 +1,334 @@
...
+ocd-division/country:mx/state:mo/fed:2,II Federal Electoral District of Morelos
+ocd-division/country:mx/state:mo/fed:3,III Federal Electoral District of Morelos
+ocd-division/country:mx/state:mo/fed:4,IV Federal Electoral District of Morelos
+ocd-division/country:mx/state:mo/fed:5,V Federal Electoral District of Morelos
+ocd-division/country:mx/state:na,Nayarit
+ocd-division/country:mx/state:na/fed:1,I Federal Electoral District of Nayarit
+ocd-division/country:mx/state:na/fed:2,II Federal Electoral District of Nayarit
+ocd-division/country:mx/state:na/fed:3,III Federal Electoral District of Nayarit
+ocd-division/country:mx/state:nl,Nuevo León
+ocd-division/country:mx/state:nl/fed:1,I Federal Electoral District of Nuevo León
```
Comment (ccongchen) [inline identifiers/country-mx.csv]: Update to "New Leon"?

**Comment by jdmgoogle** [comment]
Okay. I'm going to merge as-is, and @jpmckinney or @djbridges can yell at me later if that's not kosher. :)

**Comment by jpmckinney** [comment]
You can add a `name_es` column that includes the original Spanish names. For `name`, the translations to English should reflect real usage. For example, "Mexico" is very common (instead of "México") and indeed the title of the corresponding Wikipedia page. On the other hand, I've never heard "New Leon" and indeed the title of the corresponding Wikipedia page is "Nuevo León" (with accent). I haven't looked through all translations/transliterations, but they should not be too "creative" – they should, ideally, reflect practice, as demonstrated by, for example, Wikipedia page titles.

**Comment by jpmckinney** [comment]
Looking at just the last two commits, it looks like only "México" -> "Mexico" was changed, so the English names seem fine.

**Pull request merged**